### ISSUE TITLE
Fail to build pacman package when MSYS="winsymlinks=lnk"

### ISSUE THREAD

**Issue by fracting** [Alexpux/Cygwin#14]
I guess this is an invalid bug, but I still would like to hear advice.

For some reason, I use MSYS="winsymlinks=lnk" to workaround some Wine limitations on specific Linux filesystem.

However, when building pacman package on Wine, I found error like below:
http://wine-ci.org/Alexpux/MSYS2-packages/1

```
error: unable to create symlink lib/libalpm/util-common.c (Bad address)
error: unable to create symlink lib/libalpm/util-common.h (Bad address)
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'
```

I retested on Windows, set MSYS="winsymlinks=lnk" in cmd before starting msys2_shell.bat, and I can confirm the same error.

Any thoughts?

**Comment by fracting** [comment]
Update: this is the original reason I switch to MSYS="winsysmlinks=lnk" https://bugs.wine-staging.com/show_bug.cgi?id=403

**Comment by elieux** [comment]
Are you really using `winsymlinks=lnk`, or is that a typo? The correct one is `winsymlinks:lnk` as per https://cygwin.com/cygwin-ug-net/using-cygwinenv.html.

**Comment by fracting** [comment]
Yes, I'm really using `winsymlinks=lnk`. After double check, I found `winsymlinks=lnk` works the same as `winsymlinks:lnk`, work in the same way and broken in the same way. It might not be supposed to work, just pure luck :)

**Comment by fracting** [comment]
Update:

I've tweak wine-ci.org to workaround the limitation, no need to rely on `winsymlinks:lnk` anymore in custom Drone CI server. I'm still interesting in this issue because it might affect Travis CI users who run Wine to test msys2, but it is not high priority for me now.
